packages/material-ui-based-components/src/components/Slider.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import { withRuleEngine } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { Box, FormHelperText, InputLabel, Slider } from '@mui/material';
+
+const SliderComponent = (props: PROPS) => {
+    const { isError, name, label, required, description, enabled,
+        maximum, minimum, errorMessage, id, value, step }
+        = props;
+
+    function valuetext(value: number, label: any) {
+        return `${value}-${label.value}`;
+    }
+
+    const changeHandler = useCallback((event: any) => {
+        props.dispatchChange(event.target.value);
+    }, [props.dispatchChange]);
+
+    const blurHandler = useCallback((event: any) => {
+        props.dispatchBlur(event.target.value);
+    }, [props.dispatchBlur]);
+
+    const focusHandler = useCallback((event: any) => {
+        props.dispatchFocus(event.target.value);
+    }, [props.dispatchFocus]);
+
+    return (
+        <Box>
```

Comment:
can we use formcontrol?

Reply:
No, on using FormControl, the layout is distorted and label overlaps over the slider component.